essence-of-live-coding-frp-test/LICENSE

```diff
@@ -0,0 +1,30 @@
+Copyright (c) 2022, Ivan Perez, Manuel Bärenz, and Miguel Negrão
```

Comment:
Did you ask Ivan, or are you adding him because of his authorship of `yampa-test`?

Reply:
I didn't ask, I'm adding due to his authorship of yampa-test and bearriver.